CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.h

```diff
@@ -72,12 +72,18 @@ namespace Kartaclysm
 		bool m_bSuspended;
 
 	private:
+		struct RacerResults
+		{
+			std::string m_strRacerId;
+			float m_fRaceTime;
+		};
+		std::vector<RacerResults> m_vRaceResults;
```

Comment:
Why would it be a map? A vector allows us to implicitly store the position, and if more information needs to be added to the results (say a fastest individual lap time, or if we were to ever incorporate some sort of points system for performing certain actions in a race), it can easily just be added to the struct.  I don't see what problem using a map solves.